packages/docgen/coverage.md

```diff
@@ -32,7 +32,7 @@ These either happen in private API, aren't relevant, or are pending of decission
 - [ ] `@constant` packages/editor/src/hooks/align.js
 - [ ] `@constructor` packages/edit-post/src/hooks/components/media-upload/index.js
 - [ ] `@inheritdoc` packages/edit-post/src/components/meta-boxes/meta-boxes-area/index.js
-- [ ] `@private` packages/editor/src/components/rich-text/index.js
+- [x] `@private` packages/editor/src/components/rich-text/index.js
```

Comment:
Personally I don't think this should exist as a file in the filesystem / repository. It should be a tracking issue in GitHub.

Reply:
Some have already a corresponding issue. I'm working now on either ignore or create an issue for the rest. We can remove it in this PR.

Reply:
Finished reviewing this and created a bunch of new issues.

Reply:
> Some have already a corresponding issue. I'm working now on either ignore or create an issue for the rest. We can remove it in this PR.

I didn't see the final sentence here 'til now, but is a separate task anyways. I created a pull request at #15196.